src/main/java/com/geespring/marketapi/controller/CartController.java

```diff
@@ -0,0 +1,70 @@
+package com.geespring.marketapi.controller;
+
+import com.geespring.marketapi.dto.cart.CartProductDTO;
+import com.geespring.marketapi.model.Cart;
+import com.geespring.marketapi.model.Product;
+import com.geespring.marketapi.service.CartService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "cart")
+@RequiredArgsConstructor
+public class CartController {
```

Comment:
We need to fix all the hard-coded user IDs here. Best to retrieve a previously created session token from a header, use it to look up the `User`, and add to the `Cart` of that user.

Reply:
The system is prepared for the sessions as there are no sessions yet, once sessions are created i will replace the magic numbers with retrieved information from sessions